### ISSUE TITLE
Single Link navigation (including tray)

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2263]
It's impossible to see where your keyboard focus is, navigating the banner on top of the iframe. You can trigger the "show record details" and "hide record details" if you take a stab in the dark and hit enter when you guess your focus is there, but yeah.

The "edit link details" text, clickable, is not a button or link and doesn't appear in the tab order:
`<span class="_verbose _toDesktop"> link details</span>`

When the tray is opened, keyboard focus should be set inside there. We should consider marking this up like a modal, so you can't accidentally start tabbing around behind it, or mark it up like a dropdown, so it disappears if you tab out of it. I'd also expect it to close when I hit escape, or when I click outside of the tray.

The "upload" and "cancel" buttons in the tray can't seem to be tabbed to (despite them being button elements: investigate). They also have no hover styles (can't test focus styles)

The "download archive" and "mark as public/private" buttons have no focus styles either. The delete button's focus style is pretty subtle.
